src/main/java/org/mozilla/zest/core/v1/ZestAssignFromElement.java

```diff
@@ -0,0 +1,253 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.zest.core.v1;
+
+import net.htmlparser.jericho.Element;
+import net.htmlparser.jericho.Source;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ZestAssignFromElement extends ZestAssignment {
+
+    private boolean filterElementName;
+    private String elementNameFilter;
+
+    private boolean filterAttributeValue;
+    private String attributeNameFilter;
+    private String attributeValueFilter;
+    private int attributeValueFilterFlags;
+    transient private Pattern attributeValueFilterPattern;
```

Comment:
Could you swap the modifiers? (To follow the JLS and also the rest of the code.)